roles/postgres/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: 'Install postgresql-server'
+  become: true
+  package:
+    name:
+      - 'postgresql'
+      - 'postgresql-server'
+      - 'python-psycopg2' # required by postgresql_* modules
+    state: 'latest'
+
+- name: 'Initialize postgresql'
+  become: true
+  command: 'postgresql-setup initdb'
+  register: command_result
+  failed_when: command_result.rc != 0 and 'Data directory is not empty!' not in command_result.stdout
```

Comment:
I think the recommended way to check for a non-zero exitstatus would be:
```
failed_when: (command_result | failed) and [...]
```

Reply:
I wonder if I like the version of @ehelms with ```creates``` better or this one.

Reply:
I think I like `creates` too.